src/include/btmem.h

```diff
@@ -239,6 +242,7 @@ struct __wt_ovfl_reuse {
 #define WT_HS_CONFIG                                                   \
     "key_format=" WT_HS_KEY_FORMAT ",value_format=" WT_HS_VALUE_FORMAT \
     ",block_compressor=" WT_HS_COMPRESSOR                              \
+    ",internal_page_max=16KB"                                          \
```

Comment:
Just want to check that this change can fix most of the cases where the default page is 4K, in case if the user changes the default page size of both internal and leaf for a particular btree to accommodate bigger keys, only for those tables we will create the overflow keys in the history store table. The max key size depends on the page size.

Reply:
Our primary use case is default MongoDB, it is very rare for a MongoDB user to have custom-sized internal/leaf pages. But the default index settings, where the keys for index could be unboundedly large, will form a keystring that forms the part of the history store key. Earlier there used to be a limit of 1k for the index key size, which has been lifted because occasionally users will hit it. Our goal with raising internal page size is to accommodate those indexes. It would still be possible to create overflow keys, but much more unlikely with a larger internal page size.

Reply:
Yes, that helps. I just want to know how frequent is the scenario of increasing page sizes.